src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/SystemCommandLine/Commands.xml

```diff
@@ -20,6 +20,26 @@
       <SeeAlso Title="sources command (nuget.exe)" Url="/nuget/reference/cli-reference/cli-ref-sources" />
     </Noun>
   </Verb>
+  <Verb Name="disable">
```

Comment:
Team review: Please add guide on how to update boilerplate C# code from XML.

Reply:
Added a line in this xml file.